cocoa-foundation/src/foundation.rs

```diff
@@ -229,7 +229,12 @@ pub trait NSProcessInfo: Sized {
         msg_send![class!(NSProcessInfo), processInfo]
     }
 
+    unsafe fn systemUptime(_: Self) -> NSTimeInterval {
+        msg_send![class!(NSProcessInfo), systemUptime]
```

Comment:
Why does this need a default implementation instead of just relying on the implementation below like the other functions? i.e. why isn't this implemented more like operatingSystemVersion etc.

Reply:
It doesn't need a default implementation. We can actually have the same implementation as `operatingSystemVersion` , good point.